scripts/test.ts

```diff
@@ -201,12 +202,13 @@ glob.sync('packages/**/*.spec.ts')
     console.error(`Invalid spec file name: ${path}. You're using the old convention.`);
   });
 
-// Run the tests.
-const allTests =
-  glob.sync('packages/**/*_spec.ts')
-    .map(p => relative(projectBaseDir, p))
-    .filter(p => !/schematics_cli\/schematics\//.test(p));
+export default function (args: ParsedArgs) {
+  const regex = `packages/${args.glob ? `**/${args.glob}/` : ''}**/*_spec.ts`;
```

Comment:
I think this could be easier to follow:

```typescript
let regex = ‘packages/**/*_spec.ts’;
if (args.glob) {
  regex = `packages/**/${args.glob}/**/*_spec.ts`;
}
```

What do you think?